n3fit/src/n3fit/model_gen.py

```diff
@@ -88,7 +89,8 @@ def _generate_loss(self, mask=None):
                 invcovmat_matrix, self.data, mask, covmat=covmat_matrix, name=self.name
             )
         elif self.positivity:
-            loss = losses.LossPositivity(name=self.name, c=self.multiplier)
+            alpha = 0.0 if self.polarized else 1e-7  # ELU -> RELU for Polarized Fits
```

Comment:
Because what we want is $\mathcal{O}_{\mathrm{POS}} \equiv |\Delta  f_i(x,Q^2)| - f_i(x,Q^2)  \le 0$ and therefore negative values shouldn't contribute to the penalty.

Reply:
But having ELU just avoids a sharp cut at 0.0.
If the difference is relevant I'd be worried... but in any case, the way to go would be making the positivity_alpha or whatever a runcard parameter.

Reply:
But this is exactly what you want in the polarized case and seen to be empirically improve the convergences of the fits. But I agree that the best is to allow for such parameter to be defined in the runcard (this would requires a few modifications though).

Reply:
If you do the fit with the default 1e-7 it doesn't work?

The idea of ELU and 1e-7 is that it was a numerical 0 so it doesn't affect the loss/chi2 but at the same time avoids a sharp cut at 0. We are using float32 for the fit so 1e-7 should be irrelevant.

Reply:
This actually I should try again. Maybe things have changes with all the fixes and refactoring.

Reply:
I turned out that this now works (there might have been some combinations of issues before). I have now removed it completely and the need to pass `fitbasis` everywhere.